src/client.ts

```diff
@@ -613,9 +613,9 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
    *
    * @param softphoneParams participant information for initiating a softphone session. See IStartSoftphoneSessionParams for more details.
    */
-  async startSoftphoneSession (softphoneParams: IStartSoftphoneSessionParams): Promise<{id: string, selfUri: string}> {
-    softphoneParams.sessionType = SessionTypes.softphone;
-    return this.sessionManager.startSession(softphoneParams);
+  async startSoftphoneSession (softphoneParams: Omit<IStartSoftphoneSessionParams, 'sessionType'>): Promise<{ id: string, selfUri: string }> {
+    (softphoneParams as IStartSoftphoneSessionParams).sessionType = SessionTypes.softphone;
```

Comment:
This was a local change I had floating around. Need to omit the need for `SessionType` when starting a call from the sdk.